server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1486,234 +1495,6 @@ public RemoteStateTransferException(String errorDesc, Throwable cause) {
         }
     }
 
-    /**
```

Comment:
Refactored all following clean up methods to new `RemoteClusterStateCleanupManager.java` class